### PR TITLE
Add StackBlitz

### PR DESCRIPTION
**Pull request by patak-cat**
First of all, congrats on the repercussions of the OSS Pledge initiative. It is great to see all the OSS sustainability discussions that it has sparked.

At [StackBlitz](https://stackblitz.com), we have been backing OSS infrastructure projects (Vite, Vitest, Volar, Wasm) for the past years. We're excited to join the OSS Pledge with you.

We have published a blog post explaining our contributions: [StackBlitz Joins OSS Pledge Post](https://blog.stackblitz.com/posts/stackblitz-joins-osspledge/).

We'll be sharing it after we check all looks good and this PR is merged.

I think later on, maybe we could also discuss if OSS maintainers that are hired as full-time employees by companies to work on external projects could be also accounted (this is my case working in Vite, and also Ari Perkkio working in Vitest). For now, we aren't including us for the pledge, and only detailing StackBlitz contributions to external maintainers.

Thanks again for pushing this initiative forward!

Note: here is a SVG for the [StackBlitz Logo](https://viteconf.org/images/logos/stackblitz.svg), let me know if you need other assets.

### CODE REVIEW

**Review by vladh — CHANGES_REQUESTED** [review]
In the blog post, could you please change references to “OSS Pledge” to say “Open Source Pledge”? The name of the initiative is “Open Source Pledge”, while “osspledge.com” is the domain”. Should have been “opensourcepledge.com” if you ask me. 😛

**Comment by vladh** [comment]
Hey @patak-dev, it's great to have you on board! 🎉  And thanks so much for the kind words.

Other than the comment I left above, everything looks good to me!

You raised the question of whether the time of developers hired by a company to contribute to Open Source projects should be counted as part of the Pledge. We've decided that this time will not count towards the cash amount, but we do have a field for this, `monetaryValueOfTime`. Feel free to add these contributions there if you want.

**Comment by patak-dev** [comment]
> Hey @patak-dev, it's great to have you on board! 🎉 And thanks so much for the kind words.

Thanks!

> Other than the comment I left above, everything looks good to me!

I'll update the blog post.

> You raised the question of whether the time of developers hired by a company to contribute to Open Source projects should be counted as part of the Pledge. We've decided that this time will not count towards the cash amount, but we do have a field for this, `monetaryValueOfTime`. Feel free to add these contributions there if you want.

As it is currently shown in the page (for example here https://osspledge.com/members/frontend-masters/), this field reads more related to the employee costs to organize and distribute the funds. In my case, it is unrelated to this work. I understand that full-time employees working for external projects aren't easy to define and include in the pledge though. I also don't think the actual monetary cost here could be accounted. If added later, maybe it could be an equivalence that is fixed across all companies. Like one full-time employee counting as a fixed $X per year for everybody. But I completely agree with your current decision of keeping things simple for now. Again, thanks a lot for organizing this initiative!

**Comment by vladh** [comment]
> As it is currently shown in the page (for example here https://osspledge.com/members/frontend-masters/), this field reads more related to the employee costs to organize and distribute the funds.

Oh, I see! The field is intended for employee time spent contributing to Open Source projects. Maybe we need to clarify this — I've opened #117.

**Comment by patak-dev** [comment]
Thanks @vladh!

We have updated the blog post using "Open Source Pledge": https://blog.stackblitz.com/posts/stackblitz-joins-osspledge/

**Review by vladh — APPROVED** [review]

**Comment by vladh** [comment]
Thank you! Waiting for a second review then we're good to go.

**Comment by vladh** [comment]
@patak-dev @chadwhitacre Oops, not sure how we missed this, but `urlLearnMore` has the logo, and `urlSquareLogoWithBackground` has the “learn more” link. Reverting.

**Comment by patak-dev** [comment]
Oh, sorry about that! We've corrected it. @vladh, should I send a new PR or you'll revert the revert?

**Comment by vladh** [comment]
All done! :tada: https://osspledge.com/members/stackblitz/

**Comment by patak-dev** [comment]
Thanks a lot! We're very excited to join the pledge! 🎉 ❤️

**Comment by patak-dev** [comment]
Here is a complete SVG set


https://developer.stackblitz.com/img/logo/stackblitz-bolt-blue.svg


https://developer.stackblitz.com/img/logo/stackblitz-bolt-white.svg


https://developer.stackblitz.com/img/logo/stackblitz-logo-black_blue.svg


https://developer.stackblitz.com/img/logo/stackblitz-logo-black.svg


https://developer.stackblitz.com/img/logo/stackblitz-logo-white_blue.svg


https://developer.stackblitz.com/img/logo/stackblitz-logo-white.svg

**Comment by vladh** [comment]
Perfect, thank you! 🎉

**Pull request merged**
